tools/AzureMarkdownRewriterTool/Program.cs

```diff
@@ -122,32 +122,98 @@ private static RewriterToolArguments ParseRewriterToolArgumentsFile(string repos
             }
         }
 
-        private static Dictionary<string, AzureVideoInfo> ParseAzureVideoFile(string argsFilePath)
+        private static Dictionary<string, AzureVideoInfo> ParseAzureVideoFile(string argsFilePath, bool isMigration)
         {
             if (!File.Exists(argsFilePath))
             {
                 Console.WriteLine("Can't find video mapping info file. Skip transform step for video.");
                 return null;
             }
 
-            var argsContent = File.ReadAllText(argsFilePath);
+            var videoInformationContent = File.ReadAllText(argsFilePath);
             try
             {
-                var azureVideoInfoList = JsonConvert.DeserializeObject<List<AzureVideoInfo>>(argsContent);
                 var azureVideoInfoMapping = new Dictionary<string, AzureVideoInfo>();
-                foreach(var azureVideoInfo in azureVideoInfoList)
+                if (isMigration)
+                {
+                    var azureVideoRawInfoMapping = new Dictionary<string, AzureVideoDataItem>();
+                    var azureVideoRawInformation = JsonConvert.DeserializeObject<AzureVideoRawInformation>(videoInformationContent);
```

Comment:
JsonUtility.Deserialize 

---
In reply to: [73273690](https://github.com/dotnet/docfx/pull/559#discussion_r73273690) [](ancestors = 73273690)